tests/tests_suite_test.go

```diff
@@ -407,7 +415,9 @@ var _ = BeforeSuite(func() {
 	setupApiClient()
 	strategy.Init()
 
-	clusterMachineType = retrieveQemuMachineType()
+	nodeArchitecture = retrieveNodeArchitecture()
```

Comment:
Tbh, I'm not sure this will bite us in the future. It assumes the arch of all nodes by looking at the first node only.

Reply:
The primary concern here is that the s390x templates are provided through a suffix. As I mentioned earlier, if the common templates were delivered as three separate files—one generic, one for amd64, and one for s390x—the tests would become architecture-independent.
The goal is to introduce an intermediary step. Once the multi-architecture common templates are delivered to the ssp-operator, we can simplify the code while minimizing the risk of breaking anything.